components/builder-worker/src/runner/mod.rs

```diff
@@ -162,6 +163,10 @@ impl Runner {
     }
 
     pub fn run(mut self) -> Job {
+        if let Some(err) = util::validate_integrations(&self.workspace).err() {
+            error!("failed to validate integrations err={:?}", err);
+            return self.fail(net::err(ErrCode::INVALID_INTEGRATIONS, "wk:run:5"));
```

Comment:
We should add a new code for the validation failure -  wk:run:5 is used for overall build failure

Reply:
Ack! I did this part way too fast, good catch. I'll make this a `:7` and re-push.